### ISSUE TITLE
missing return value in updateConf function (applyRules.js)

### ISSUE THREAD

**Issue by calpa** [ivarprudnikov/rjsf-conditionals#64]
In order to update conf, there is a `newValues` return in the `updateConf` function.

```js
handleChange(formChange) {
        const { formData } = formChange;
        const { onChange } = this.props;
        if (!deepEquals(formData, this.state.formData)) {
          this.updateConf(formData, (newValues) => {
            if (onChange) {
              let updChange = Object.assign({}, formChange, newValues);
              onChange(updChange);
            }
          });
        } else {
          onChange && onChange(formChange);
        }
      }
```

But Line 120 is missing the return value, which makes it wrong.

**Comment by ivarprudnikov** [comment]
> In order to update conf
Assume it is `updateConf(formData, changeHandler)`, no?

> there is a `newValues` return
Which file and which line is being referred?

> But Line 120 is missing the return value
If it is `runRules(formData).then(...)` then not sure why would it need a return value? 

> which makes it wrong
In what way?

As usual a small example reproducing an issue would be helpful (jsfiddle, codepen, etc.)

**Comment by calpa** [comment]
@ivarprudnikov while I am trying to create a sandbox,
another problem occurs...

```
TypeError
(0 , require.default) is not a function
```

You can check this sandbox
https://codesandbox.io/s/happy-swartz-1vo5v?file=/src/App.js

**Comment by calpa** [comment]
Also, I find that `CacheControl/json-rules-engine`,
engine.run([Object facts], [Object options]) -> Promise ({ events: Events, almanac: Almanac})
it returns an object instead of events.

```js
let res = engine.run(formData).then((events) => { // <- here
    events.forEach((event) =>
      execute(event, schemaCopy, uiSchemaCopy, formDataCopy, extraActions)
    );
  });
```

Therefore, the above code throws error.

**Comment by ivarprudnikov** [comment]
@calpa I've created a sandbox you can fork:
https://codesandbox.io/s/rjsf-conditions-basic-example-frpty

You can see in sandbox I've imported `umd` (`/dist/rjsf-conditionals.js`) dependency instead of default one which lives in `/dist/es/index.js`.

Please fork the sandbox and attempt at reproducing an issue which was mentioned in the beginning.

**Comment by calpa** [comment]
@ivarprudnikov I had reproduce the issue I mentioned above.

https://codesandbox.io/s/angry-ellis-1d4dw

**Comment by ivarprudnikov** [comment]
I see it happens when using `json-rules-engine` but not on `json-rules-engine-simplified`.

**Comment by calpa** [comment]
Yes, the main reason is engine.run([Object facts], [Object options]) -> Promise ({ events: Events, almanac: Almanac}), it returns an object instead of events.

**Comment by ivarprudnikov** [comment]
@calpa I've released v1.0.1 with a fix. If you find any more problems please open another issue.
